builtin/logical/pki/path_roles.go

```diff
@@ -490,11 +489,11 @@ type roleEntry struct {
 	UseCSRCommonName      bool   `json:"use_csr_common_name" structs:"use_csr_common_name" mapstructure:"use_csr_common_name"`
 	KeyType               string `json:"key_type" structs:"key_type" mapstructure:"key_type"`
 	KeyBits               int    `json:"key_bits" structs:"key_bits" mapstructure:"key_bits"`
-	MaxPathLength         *int   `json:",omitempty" structs:",omitempty"`
+	MaxPathLength         *int   `json:"max_path_length,omitempty" structs:"max_path_length,omitempty" mapstructure:"max_path_length"`
```

Comment:
Can't do this! If you change the JSON name here, when we deserialize next time it won't match and the value will revert to a default instead of keeping the real value. Leave JSON alone, only set `structs` and `mapstructure`.